pkgs/games/am2rlauncher/default.nix

```diff
@@ -0,0 +1,113 @@
+{ lib,
+  buildDotnetModule,
+  pkgsi686Linux,
+  glibc,
+  gtk3,
+  libappindicator,
+  webkitgtk,
+  e2fsprogs,
+  libnotify,
+  libgit2,
+  openssl,
+  xdelta,
+  file,
+  busybox,
+  openjdk,
+  steam-run,
+  patchelf,
+  fetchFromGitHub,
+  makeDesktopItem }:
```

Comment:
I don't usually see the function inputs formatted like this.
Use something like `alejandra` to format the derivation.